Docs/mc_definition.tex

```diff
@@ -174,6 +169,7 @@ \subsection{Scripts}
 \subsubsection{Geographical Information:  {\tt geo.py fg\_action --arguments}}
 
 Has various plotting/printing options for station information.  {{\tt fg\_action} is the "foreground action", that is the locations that will get printed and shown on top if plotting.
```

Comment:
Double quotes don't work properly in latex -- you need to use the backtick and single quotes to get them to go the correct direction.